client/src/pages/explorer.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+//Remove the next line when backend data is available.
+import places from "../database/places";
+import Grid from "@material-ui/core/Grid";
+import Container from "@material-ui/core/Container";
+import Favorite from "@material-ui/icons/Favorite";
+import Checkbox from "@material-ui/core/Checkbox";
+import { withStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+  pageContainer: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    marginTop: 20,
+  },
+  paperContainer: {
+    height: 250,
+    width: 195,
+    borderRadius: 10,
+    backgroundRepeat: "no-repeat",
+    backgroundSize: "cover",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "flex-end",
+    overflow: "hidden",
+  },
+  title: {
+    margin: 5,
+    textAlign: "center",
+  },
+  customCheckBoxRoot: {
+    width: "5px",
+    height: "5px",
+  },
+});
+
+const CustomSmallerCheckBox = withStyles({
+  root: {
+    "& .MuiSvgIcon-root": { width: "15px", height: "15px" },
+  },
+})(Checkbox);
+
+function FavoriteCheckBox() {
+  const classes = useStyles();
+  const [checked, setChecked] = useState(false);
+  return (
+    <div>
+      <CustomSmallerCheckBox
+        checked={checked}
+        onChange={(e) => setChecked(e.target.checked)}
+        icon={<Favorite style={{ color: "white" }} />}
+        checkedIcon={<Favorite style={{ color: "orange" }} />}
+        classes={{ root: classes.customCheckBoxRoot }}
+      />
+    </div>
+  );
+}
+
+const ExplorerPage = () => {
+  const classes = useStyles();
+  return (
+    <Container className={classes.pageContainer}>
+      <Typography variant="h4" className={classes.title}>
+        Explore destinations
+      </Typography>
+      <Typography
+        variant="subtitle2"
+        className={classes.title}
+        style={{ color: "#c5bec4", fontSize: 12 }}
+      >
+        World's top destinations to explore
+      </Typography>
+      <Grid
+        container
+        spacing={2}
+        justify="center"
+        style={{ maxWidth: "75%", marginTop: 32 }}
+      >
+        {places.map((place) => (
+          <Grid item>
+            <div
+              className={classes.paperContainer}
+              style={{
+                backgroundImage: `linear-gradient(to bottom, transparent, rgba(52, 52, 52, 0.63)), url(${place.imageUrl})`,
+              }}
+            >
+              <div
+                style={{
+                  display: "flex",
+                  height: "15%",
+                  borderTop: "1px solid #a9a9a9",
+                  padding: 10,
+                  justifyContent: "space-between",
+                }}
```

Comment:
We can move a lot of these up to the `makeStyles`. You can use the [classNames](https://www.npmjs.com/package/classnames) package to combine multiple styles if you need.